### DISCUSSION TITLE
runr.app - utility to run dynamic codes

### DISCUSSION THREAD

**Discussion by hdrz** [OpenRadioss/OpenRadioss#2417] [Show and tell]
Hi everyone,

I work with dynamic codes in my day job, and over the years I developed an application to help me pre-, post-, and process jobs more easly:

[runr.app](https://runr.app/)

It’s an internal tool at the moment, and I’m trying to decide what to do with it going forwards:
  1. Release as commercial app. 
  2. Release as open source app, and hope for funding thru donations. 

I’m leaning towards open source, since this way the app can benefit from the knowledge of others as well. 
Oh yeah, OpenRadioss support is in the pipeline and will be ready soon. 

Would any of you be interested in using such app? Maybe contributing?

Thanks!

**Comment by MarianBulla** [comment]
Hi @hdrz,

thank you very much for your interest in OpenRadioss and in the effort you spent in developing your app.

I had a look at your app and website, and it looks great !

Maybe a demo of the app would be more helpful ?

A good starting point could be our YouTube channel.
I'm sure, with a ~10minutes demo video you can get some more attention and suggestions for further proceedings.

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by hdrz** [comment]
Thank you @MarianBulla for the kind words. 

Making a video of using the software is a great suggestion!  I have to admit that I never done such a thing, and it will take me some time to get going…

Anyhow, let me add some details about it. The software is used just like the tcl/tk gui, but it has more functionality, and I tried to make the gui as clear and easy as possible, at least for me…

It is written in object pascal, and the gui is made with [lazarus](https://www.lazarus-ide.org/). The framework is cross-platform, and I took care not to use platform specific widgets, so it works well on linux and windows. 

Development started years ago with ls-dyna in mind, then I added radioss, and now openradioss. I want to see it used more, as I think others will benefit from it as well. 

Thanks for reading!
